config/http.conf

```diff
@@ -9,15 +9,42 @@ http {
   client_max_body_size  400m;
   client_body_timeout   300s;
 
+  # If we receive Upgrade, set Connection to "upgrade"; otherwise, delete any
+  # Connection header that may have been passed to this server
+  map $http_upgrade $proxy_connection {
+    default upgrade;
+    '' close;
+  }
```

Comment:
Granted, I'm not sure about this but I figured I'd put it here and we discuss. See ...
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Connection

Reply:
After reading [this blog post](https://ma.ttias.be/enable-keepalive-connections-in-nginx-upstream-proxy-configurations/) I don't think we need to do this. We set `proxy_set_header Connection ""` because the client can set it to `close` for a `keepalive` connection ... and we want `keepalive` (which is the default for `HTTP/1.1`) so that `nginx` can reuse existing `TCP` connections without creating new ones